helm/basex-validator/templates/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "basex-validator.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "basex-validator.name" . }}
+    helm.sh/chart: {{ include "basex-validator.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
explicitly supporting multiple replicas?

Reply:
Not intentionally, probably an artefact of the chart that I copied it from.